flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/BootstrapTools.java

```diff
@@ -185,8 +185,8 @@ public static WebMonitor startWebMonitorIfConfigured(
 
 		// this ensures correct values are present in the web frontend
 		final Address address = AkkaUtils.getAddress(actorSystem);
-		config.setString(ConfigConstants.JOB_MANAGER_IPC_ADDRESS_KEY, address.host().get());
-		config.setString(ConfigConstants.JOB_MANAGER_IPC_PORT_KEY, address.port().get().toString());
+		config.setString(JobManagerOptions.ADDRESS, address.host().get());
+		config.setInteger(JobManagerOptions.PORT, Integer.parseInt(address.port().get().toString()));
```

Comment:
I don't think we can cast it to Integer since it is a scala Int actually...let's keep it as it is.